website/src/components/footer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Footer = () => <footer>
+  <span >© 2020-2024 Björn Büttner and contributors.</span>
+  <ul>
+    <li>
+      <a href="#">About</a>
+    </li>
+    <li>
+      <a href="#">Privacy Policy</a>
+    </li>
+    <li>
+      <a href="#">Licensing</a>
```

Comment:
License goes to /license and uses the content of LICENSE in the root of the repository